Phase Kickback Tutorial (#777)
PennyLaneAI/qml: demos_getting-started.rst

    :description: :doc:`demos/tutorial_qft_arithmetics`
    :tags: qft qc short

.. gallery-item::

>     :tooltip: Implement a quantum lock using phase kickback.

Reviewer: ```suggestion
    :tooltip: Use phase kickback to create an unbreakable quantum lock.
```

>     :figure: demonstrations/phase_kickback/Phase_Kickback.png

Reviewer: This would probably need to be replaced with a fancy PL-style illustration at publication.

Author: The thumb nail is PL-style, I took it from the codebook.

Reviewer: Indeed, but in the meantime we have actually gotten a professional designer on board, who will polish it up to be used in thumbnail form. :) This comment is for you to be aware and for Guillermo to not forget.
